libraries/chain/include/steemit/chain/comment_object.hpp

```diff
@@ -37,79 +37,20 @@ namespace steemit { namespace chain {
          }
    };
 
-   /**
-    *  Used to track the trending categories
-    */
-   class category_object : public object< category_object_type, category_object >
-   {
-      category_object() = delete;
-
-      public:
-         template< typename Constructor, typename Allocator >
-         category_object( Constructor&& c, allocator< Allocator > a )
-            :name( a )
-         {
-            c( *this );
-         }
-
-         id_type        id;
-
-         shared_string  name;
-         share_type     abs_rshares;
-         asset          total_payouts = asset(0, SBD_SYMBOL);
-         uint32_t       discussions = 0;
-         time_point_sec last_update;
-   };
-
-   struct by_name;
-   struct by_rshares;
-   struct by_total_payouts;
-   struct by_last_update;
-   typedef multi_index_container<
-      category_object,
-      indexed_by<
-         ordered_unique< tag< by_id >, member< category_object, category_id_type, &category_object::id > >,
-         ordered_unique< tag< by_name >, member< category_object, shared_string, &category_object::name >, strcmp_less >,
-         ordered_unique< tag< by_rshares >,
-            composite_key< category_object,
-               member< category_object, share_type, &category_object::abs_rshares>,
-               member< category_object, category_id_type, &category_object::id >
-            >,
-            composite_key_compare< std::greater< share_type >, std::less< category_id_type > >
-         >,
-         ordered_unique< tag< by_total_payouts >,
-            composite_key< category_object,
-               member< category_object, asset, &category_object::total_payouts>,
-               member< category_object, category_id_type, &category_object::id >
-            >,
-            composite_key_compare< std::greater<asset>, std::less< category_id_type > >
-         >,
-         ordered_unique< tag< by_last_update >,
-            composite_key< category_object,
-               member< category_object, time_point_sec, &category_object::last_update>,
-               member< category_object, category_id_type, &category_object::id >
-            >,
-            composite_key_compare< std::greater< time_point_sec >, std::less< category_id_type > >
-         >
-      >,
-      allocator< category_object >
-   > category_index;
-
    class comment_object : public object < comment_object_type, comment_object >
    {
       comment_object() = delete;
 
       public:
          template< typename Constructor, typename Allocator >
          comment_object( Constructor&& c, allocator< Allocator > a )
-            :category( a ), parent_permlink( a ), permlink( a ), title( a ), body( a ), json_metadata( a ), beneficiaries( a )
+            :parent_permlink( a ), permlink( a ), title( a ), body( a ), json_metadata( a ), beneficiaries( a )
          {
             c( *this );
          }
 
          id_type           id;
 
-         shared_string     category;
```

Comment:
I think for backwards compatibility of comments that existed before we were use metadata tags, this field should still exist.